cmake/config.cmake

```diff
@@ -166,6 +166,7 @@ if(OMR_THR_YIELD_ALG)
 endif()
 #TODO set to disabled. Stuff fails to compile when its on
 set(OMR_THR_TRACING OFF CACHE BOOL "TODO: Document")
+set(OMR_THR_MCS_LOCKS OFF CACHE BOOL "TODO: Document")
```

Comment:
Please document the flag, just something short.

Reply:
Added a short description for the flag.